xcm/v4/transfers_dev.json

```diff
@@ -5743,20 +5743,6 @@
                 }
               },
               "type": "xcmpallet"
-            },
-            {
-              "destination": {
```

Comment:
why we remove this destination from dev?

Reply:
that destination does not work for some reason

Reply:
removed from the PROD too